### DISCUSSION TITLE
Windows Package Manager 1.6.3482

### DISCUSSION THREAD

**Discussion by ryfu-msft** [microsoft/winget-cli#3992] [Releases]
This release is the seventh stable release of Windows Package Manager 1.6 for Windows 10 (1809+) and Windows 11. Experimental features have been disabled in this release. The store certificates in this release have been updated. The current store certificates will expire in early March 2024.

The [`winget configure`](https://learn.microsoft.com/windows/package-manager/configuration/) command is now a stable feature and can be used to automatically handle the setup and configuration requirements for an ideal development environment on your Windows machine. Applying a WinGet configuration file helps with installing and managing software packages, applications, programming languages, frameworks, tools, or settings necessary for a project.

Support for package dependencies is also included in this stable release. You can also use the `winget download` command to download a package installer to your local machine. 

## What's Changed
* Include framework packages during installed packages enumeration by @yao-msft in https://github.com/microsoft/winget-cli/pull/3975
* Introduce strong and weak comparisons between installers by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/3956
* Update Store Certs by @yao-msft in https://github.com/microsoft/winget-cli/pull/3968

**Full Changelog**: https://github.com/microsoft/winget-cli/compare/v1.6.3421...v1.6.3842

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.6.3842'>Windows Package Manager 1.6.3842</a>.</em>

**Comment by Two38** [comment]
it is 1.6.3482 (not 3842)

**Reply by yao-msft** [reply]
Thanks for letting us know.

**Comment by tedy-art** [comment]
please add some installation steps

**Comment by dogdogbone** [comment]
...maxbundle does not install on my Win10 despite this advice:
https://www.computerworld.com/article/3684171/winget-the-best-way-to-keep-windows-apps-updated.html

After downloading Windows cannot open or run it.
